sdk/olm_api_client/v1/mock_client.py

```
@@ -0,0 +1,107 @@
+    "I'm sorry, but could you be more specific about what you're looking for?",
+]
+
+
+class MockOlmClientV1:
+    """
+    A high-fidelity mock client that simulates v1 API behavior.
+    """
+
+    def __init__(
+        self,
+        api_url: str = None,
```

Ruff (0.12.2)

Unused method argument: `api_url`

(ARG002)